PokeAlarm/WebhookStructs.py

```diff
@@ -71,7 +75,8 @@ def pokemon(data):
             'gender': get_pokemon_gender(check_for_none(int, data.get('gender'), '?')),
```

Comment:
I don't like the idea of swapping one attribute for another like this, because if I am expecting a neutral gender yet suddenly I receive form instead!